docs/src/modules/components/AppSearch.js

```diff
@@ -28,7 +28,7 @@ function AlgoliaStyles() {
     <GlobalStyles
       styles={(theme) => {
         return {
-          '.algolia-autocomplete': {
+          '.algolia-autocomplete.algolia-autocomplete': {
```

Comment:
I've bumped the specificity here. I tried moving the `<style id="app-search"` in the _document.js, but it didn't help as emotion is injected first.